source/apply.html.haml

```diff
@@ -1,5 +1,5 @@
 ---
-title: Apply for CMO 2016
+title: Apply for CMO
```

Comment:
Since we still have the current project year displayed in the hero unit, let's interpolate here the same way.

Reply:
Interpolation didn't work here, so we'll just leave it as is for now.